pkl-cli/src/main/kotlin/org/pkl/cli/commands/FormatterCommand.kt

```diff
@@ -26,75 +25,70 @@ import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.types.enum
 import com.github.ajalt.clikt.parameters.types.path
 import java.nio.file.Path
-import org.pkl.cli.CliFormatterApply
-import org.pkl.cli.CliFormatterCheck
-import org.pkl.cli.commands.FormatterCheckCommand.Companion.grammarVersionHelp
-import org.pkl.commons.cli.commands.BaseCommand
+import org.pkl.cli.CliFormatterCommand
 import org.pkl.formatter.GrammarVersion
 
-class FormatterCommand : NoOpCliktCommand(name = "format") {
-  override fun help(context: Context) = "Run commands related to formatting"
+class FormatterCommand : CliktCommand(name = "format") {
+  override fun help(context: Context) =
+    """
+    Format or check formatting of Pkl files.
+    
+    Exit codes:
+      - 0: All files were properly formatted
+      - 1: A non-formatting error ocurred (Ex.: IO error)
+      - 11: One or more files had formatting violations
```

Comment:
For consistency sake, let's document this on our website rather than here.

We don't describe exit codes in any of our either commands.

```suggestion
```